common/djangoapps/edeos/utils.py

```diff
@@ -150,10 +149,8 @@ def validate_wallets_data(wallet_name, profitonomy_public_key):
               ```
     """
     return (
-        # At least one digit (up to 5) and one letter
-        bool(re.match('^(?=.*[0-5]$)(?=.*[a-z])', wallet_name.strip()))
```

Comment:
Why was the `At least one digit (up to 5) and one letter` check removed?
As per https://youtrack.raccoongang.com/issue/SKILLONOMY-133, 

```
Cases - expected FAILURE
ddddddddddddd // letters only, no numbers
111111111111 // numbers only, no letters
```  
(this is also mentioned in https://youtrack.raccoongang.com/issue/SKILLONOMY-164);

```
"Имя аккаунта EOS должно быть ровно 12 символов и состоять из строчных букв и цифр до 5"
```

Reply:
There also are the tests in the `edeos` app to check:
https://github.com/raccoongang/edx-platform/blob/686b0313b9cf6117089f6506177e09acfd7dc01c/common/djangoapps/edeos/tests/utils.py

Reply:
Successs case was chenged: 
```
new_case: studywallet2 or 23451study15 //letters and numbers (numbers are from 1 to 5 inclusive) order of numbers and letters doesn't matter
new_case: 123451234511 //numbers only, no latters
new_case: studywallett //letters only, no numbers
```

Reply:
tests were checked too.